acceptance/tests/pxp_module_bolt_task/task_purged_from_cache.rb

```diff
@@ -2,6 +2,9 @@
 
 test_name 'remove old task from pxp-agent cache' do
 
```

Comment:
@johnduarte This one can be killed.

This functionality is covered in unit

Reply:
@McdonaldSeanp this has also been deleted.  Thanks.